src/util.ts

```diff
@@ -272,3 +275,22 @@ export async function findWorkingNamespace(flags: any): Promise<string> {
 
   return DEFAULT_CHE_NAMESPACE
 }
+
+export function getPlatform(): string {
+  const platform: string = os.platform()
+  if (platform.startsWith('win')) {
```

Comment:
Do we really need to convert `win` to `Windows`, `darwin` to `Mac`?

Reply:
Is not necessary. I can remove it